[2023-09-13T07:38:44.017+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_data.extract_data scheduled__2023-09-03T00:00:00+00:00 [queued]>
[2023-09-13T07:38:44.075+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_data.extract_data scheduled__2023-09-03T00:00:00+00:00 [queued]>
[2023-09-13T07:38:44.077+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-09-13T07:38:44.140+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): extract_data> on 2023-09-03 00:00:00+00:00
[2023-09-13T07:38:44.163+0000] {standard_task_runner.py:57} INFO - Started process 142 to run task
[2023-09-13T07:38:44.252+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'weather_data', 'extract_data', 'scheduled__2023-09-03T00:00:00+00:00', '--job-id', '33', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag.py', '--cfg-path', '/tmp/tmpbydijz9w']
[2023-09-13T07:38:44.288+0000] {standard_task_runner.py:85} INFO - Job 33: Subtask extract_data
[2023-09-13T07:38:44.445+0000] {task_command.py:410} INFO - Running <TaskInstance: weather_data.extract_data scheduled__2023-09-03T00:00:00+00:00 [running]> on host 190733975a8f
[2023-09-13T07:38:44.661+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_data' AIRFLOW_CTX_TASK_ID='extract_data' AIRFLOW_CTX_EXECUTION_DATE='2023-09-03T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-09-03T00:00:00+00:00'
[2023-09-13T07:38:44.983+0000] {python.py:183} INFO - Done. Returned value was: None
[2023-09-13T07:38:45.017+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=weather_data, task_id=extract_data, execution_date=20230903T000000, start_date=20230913T073844, end_date=20230913T073845
[2023-09-13T07:38:45.095+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-09-13T07:38:45.140+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-09-15T11:31:19.014+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_data.extract_data scheduled__2023-09-03T00:00:00+00:00 [queued]>
[2023-09-15T11:31:19.029+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_data.extract_data scheduled__2023-09-03T00:00:00+00:00 [queued]>
[2023-09-15T11:31:19.031+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-09-15T11:31:19.109+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): extract_data> on 2023-09-03 00:00:00+00:00
[2023-09-15T11:31:19.116+0000] {standard_task_runner.py:57} INFO - Started process 367 to run task
[2023-09-15T11:31:19.160+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'weather_data', 'extract_data', 'scheduled__2023-09-03T00:00:00+00:00', '--job-id', '23', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag.py', '--cfg-path', '/tmp/tmpi7wkw3hk']
[2023-09-15T11:31:19.169+0000] {standard_task_runner.py:85} INFO - Job 23: Subtask extract_data
[2023-09-15T11:31:19.310+0000] {task_command.py:410} INFO - Running <TaskInstance: weather_data.extract_data scheduled__2023-09-03T00:00:00+00:00 [running]> on host 4af8904e631c
[2023-09-15T11:31:19.563+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_data' AIRFLOW_CTX_TASK_ID='extract_data' AIRFLOW_CTX_EXECUTION_DATE='2023-09-03T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-09-03T00:00:00+00:00'
[2023-09-15T11:31:19.875+0000] {python.py:183} INFO - Done. Returned value was: None
[2023-09-15T11:31:19.905+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=weather_data, task_id=extract_data, execution_date=20230903T000000, start_date=20230915T113119, end_date=20230915T113119
[2023-09-15T11:31:19.988+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-09-15T11:31:20.032+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
